### PR TITLE
TNO-2239: CSS updates

### PR DESCRIPTION
**Pull request by mihailistov**

### CODE REVIEW

File: app/subscriber/src/components/basic-search/BasicSearch.tsx [hunk 1]
```diff
@@ -45,8 +45,8 @@ export const BasicSearch = ({ onSearch, inHeader }: IBasicSearchProps) => {
 
   return (
     <styled.BasicSearch inHeader={inHeader}>
-      <label>SEARCH FOR: </label>
       <Row className="search-row">
+        <label>SEARCH FOR: </label>
```
Comment (kylgarmor) [inline app/subscriber/src/components/basic-search/BasicSearch.tsx:49]: If we're doing CSS updates, could you change any instances of ALL caps in these files you are touching to mixed case and then apply a CSS `text-transform:uppercase` to them please?

File: app/subscriber/src/components/basic-search/BasicSearch.tsx [hunk 2]
```diff
@@ -77,8 +77,8 @@ export const BasicSearch = ({ onSearch, inHeader }: IBasicSearchProps) => {
           Search
           <FaPlay />
         </Button>
+        <p onClick={() => navigate(`/search/advanced/${filterId}`)}>GO ADVANCED</p>
```
Comment (kylgarmor) [inline app/subscriber/src/components/basic-search/BasicSearch.tsx:80]: not your code originally I know, but a `<p` tag acting as a link is a bit funny.  Maybe it should be a `<link` instead?

File: app/subscriber/src/components/basic-search/styled/BasicSearch.tsx [hunk 3]
```diff
@@ -64,7 +63,7 @@ export const BasicSearch = styled(Row)<{ inHeader?: boolean }>`
       height: 2em;
     }
 
-    width: 85%;
+    width: calc(90% - 15px);
```
Comment (kylgarmor) [inline app/subscriber/src/components/basic-search/styled/BasicSearch.tsx:66]: 15px is curiously close to the value of 1rem. Is there a 1rem thing you are decreasing the width for here?

File: app/subscriber/src/features/content/view-content/styled/ViewContent.tsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ export const ViewContent = styled.div`
     margin-right: -1.25em;
     padding-left: 1.25em;
     padding-right: 1.25em;
-    font-size: 0.8rem;
+    font-size: 0.9rem;
```
Comment (kylgarmor) [inline app/subscriber/src/features/content/view-content/styled/ViewContent.tsx:13]: should this be `em` ?

**Review by kylgarmor — CHANGES_REQUESTED** [review]

File: app/subscriber/src/features/content/view-content/styled/ViewContent.tsx (hunk 4 above)
Comment (mihailistov) [reply app/subscriber/src/features/content/view-content/styled/ViewContent.tsx:13]: Not sure, `rem` is relative to root font size, so it'd be more consistent sizing. Whereas `em` is relative to parent font size. I'd think `rem` should be used more generally

File: app/subscriber/src/components/basic-search/styled/BasicSearch.tsx (hunk 3 above)
Comment (mihailistov) [reply app/subscriber/src/components/basic-search/styled/BasicSearch.tsx:66]: switched to 1rem

**Review by kylgarmor — APPROVED** [review]

**Pull request merged**